css/reset-css.md

```diff
@@ -0,0 +1,172 @@
+
+### Objectives
+
+**Participants will be able to:**
+
+- leave this lesson knowing this
+- and how to do this on their own
+- Even more things
+- Even more things
+
+### Specific Things To Teach
+
+- about reset.css
+
+### Materials
+
+- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
+- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
+(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+
+### Lesson
+
+Here's text about introducing something and how it works.
+
+- [Be sure to link the slideshow, video,](google.com)
+- [or other materials you expect them to follow as part of the lesson.](google.com)
+
+Build on the first information. Have apprentices guess things, do an activity, etc.
+
+Make sure to mention these things:
+
+- Things
+    - This is a sub-thing
+- More things
+- Even more things
+- Even more things
+
+
+### Common Mistakes / Misconceptions
+
+List things that apprentices might not realize, might assume at first, or should avoid.
+
+- Example
+- Example
+
+
+### Guided Practice
+#### Practice implementation of reset.css
+1. Create an HTML file with a linked CSS file (named style.css), a heading, a paragraph and an order list like this:
+```
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <h2>Lorem Ipsum</h2>
+  <p>
+    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
+  </p>
+  <ol>
+    <li>One item</li>
+    <li>two item</li>
+    <li>three item</li>
+  </ol>
+</body>
+</html>
+```
+2. Create a CSS file (named style.css), and style your HTML elements like this:
+```
+h2{
+  background-color:lightblue;
+}
+
+p{
+  background-color:lightpink;
+}
+
+ol{
+  background-color:lightgreen;
+  -moz-border-radius:5px;
+  -webkit-border-radius:5px;
+  border-radius:5px;
+}
+```
+3. Refresh your page, and then examine the webpage.
+4. You can observe that there is some white gap between each Html elements and as well as between Html element & page sides. You can also access them by the inspect tool present in your browser.
+5. Once you examine `h2` element and `ol` element through inspect tool available in your chrome browser you can clearly observe that some styles are coming from **user agent stylesheet** like this:
+```
+h2 {                                                user agent stylesheet
```

Comment:
![1](https://user-images.githubusercontent.com/33788217/56575171-b37f7600-65e2-11e9-8e07-f98c09c527ac.png)
This part of code (line 108- 116) look like this. ☝️ 
Is it the correct way it should look ? 😕

Reply:
@Kundan28 Not exactly, it shouldn't have any spaces and first line should end with `{`. Weird.

Reply:
@vsharma20v I think you understood what we meant to say.

Reply:
yes they are not correctly viewed so I will update the changes

Reply:
All specified changes are done by me